modules/UI/util/UIUtil.js

```diff
@@ -103,13 +127,13 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             'top-right': 'sw'
         };
 
-        element.setAttribute("data-i18n", "[content]" + key);
-        APP.translation.translateElement($(element));
+        $(element).each(function () {
+            var el = $(this);
+
+            el.attr("data-i18n", "[content]" + key)
+              .attr('data-tooltip', positions[position]);
```

Comment:
Regardless of the way that `positions` is defined (i.e. a function-local variable with `let`), it is practically a constant. Please move its definition of out of the function, name it in accord with best practices for constants (i.e. uppercase letters, underscores separating the words), and use `const` instead of `let`.